Neos.Workspace.Ui/Classes/Controller/WorkspaceController.php

```diff
@@ -236,20 +242,16 @@ public function createAction(
         $contentRepositoryId = SiteDetectionResult::fromRequest($this->request->getHttpRequest())->contentRepositoryId;
         $workspaceName = $this->workspaceService->getUniqueWorkspaceName($contentRepositoryId, $title->value);
 
-        $assignments = match($visibility) {
-            'shared' => WorkspaceRoleAssignments::createForSharedWorkspace($currentUser->getId()),
-            'private' => WorkspaceRoleAssignments::createForPrivateWorkspace($currentUser->getId()),
-            default => throw new \RuntimeException(sprintf('Invalid visibility %s given', $visibility), 1736343542)
-        };
-
         try {
             $this->workspaceService->createSharedWorkspace(
                 $contentRepositoryId,
                 $workspaceName,
                 $title,
                 $description,
                 $baseWorkspace,
-                $assignments
+                WorkspaceRoleAssignments::createForSharedWorkspace(
```

Comment:
to be discussed: do we need some explicit initial workspace assignments on creation or are we fine to create shared by default?